### ISSUE TITLE
Function call tip right boundary extends beyond closing parenthesis

### ISSUE THREAD

**Issue by gene-pavlovsky** [fdorg/flashdevelop#2368]
After #2324 was fixed, there is a new (minor) issue.
```haxe
	function foo(x, y) {}

	function bar() {
		foo(0, 1);
	}
```

- Put the cursor at `0`.
- Press Ctrl-Shift-1 to invoke the method call tip
- Go right character by character

Actual results: The method call tip disappears only after the cursor goes past `;` to the next line.
Expected results: The method call tip disappear after the cursor goes past `)`
Note: this is a regression most likely introduced by the fix for #2324

**Second issue** with the call tip. Modify the body of `bar` like this:
```haxe
foo(1, 2 * (3 + 4));
```
- Put the cursor at `1`.
- Press Ctrl-Shift-1 to invoke the method call tip
- Go right character by character

Actual results: The method call tip disappears after the cursor goes past `(` (the opening paren for the `3 + 4` expression).
Expected results: Method call tip still visible.
Note: this issue is also present in past versions.

**Comment by SlavaRa** [comment]
Thanks

**Comment by SlavaRa** [comment]
> Note: this is a regression most likely introduced by the fix for #2324

Nope. 5.3.3.1 has the same behavior.

**Comment by gene-pavlovsky** [comment]
I see. In that case, it was broken earlier. The (first) issue is not present in 5.3.3
